src/components/_common/image/StaticImage.tsx

```diff
@@ -4,11 +4,10 @@ import { NextImage } from './NextImage';
 
 type Props = {
     imageData: StaticImageData;
-    alt: string;
 } & React.ImgHTMLAttributes<HTMLImageElement>;
 
 export const StaticImage = (props: Props) => {
     const { imageData, alt, ...imgAttribs } = props;
```

Comment:
Alt kan kanskje for ordens skyld fjernes fra destruct  siden den ikke brukes lenger?

Reply:
Det er allerede gjort.